src/de/tests.rs

```diff
@@ -169,3 +169,43 @@ fn forgot_apostrophes() {
         _ => false,
     });
 }
+
+#[test]
+fn expected_attribute() {
+    let de: Result<String> = from_str("#\"Hello\"");
+
+    assert_eq!(de, err(ParseError::ExpectedAttribute, 1, 2));
+}
+
+#[test]
+fn expected_attribute_end() {
+    let de: Result<String> = from_str("#![enable(unwrap_newtypes) \"Hello\"");
+
+    assert_eq!(de, err(ParseError::ExpectedAttributeEnd, 1, 28));
+}
+
+#[test]
+fn invalid_attribute() {
+    let de: Result<String> = from_str("#![enable(invalid)] \"Hello\"");
+
+    assert_eq!(de, err(ParseError::NoSuchExtension("invalid".to_string()), 1, 18));
+}
+
+#[test]
+fn multiple_attributes() {
+    #[derive(Debug, Deserialize, PartialEq)]
+    struct New(String);
+    let de: Result<New> = from_str("#![enable(unwrap_newtypes)] #![enable(unwrap_newtypes)] \"Hello\"");
```

Comment:
this is not supported by the syntax highlighter, I think
Can we require the features to always come as a list in a single pragma? e.g. `#![enable(feature1, feature2)]`

Reply:
I think it is supported? It should match against the same pattern twice.